service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+  "paths" : {
+    "/service-request/status/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "summary" : "Get service request API Info",
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "service_request_no",
+          "description" : "maximum number of records to return",
+          "required" : true,
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32",
+            "minimum" : 0,
+            "maximum" : 9
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
+                  "items" : {
+                    "$ref" : "#/components/schemas/ServiceRequestSchema"
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "bad input parameter"
+          },
+          "404" : {
+            "description" : "Page Not Found"
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          },
+          "502" : {
+            "description" : "Bad Gateway Error"
+          },
+          "503" : {
+            "description" : "Service Unavailable Error"
+          }
+        }
+      }
+    },
+    "/service-request/stats/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "area",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "in" : "query",
+          "name" : "type_identifier",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
+                  "items" : {
+                    "$ref" : "#/components/schemas/AreaSchema"
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "bad input parameter"
+          },
+          "404" : {
+            "description" : "Page Not Found"
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          },
+          "502" : {
+            "description" : "Bad Gateway Error"
+          },
+          "503" : {
+            "description" : "Service Unavailable Error"
+          }
+        }
+      }
+    },
+    "/service-request/image_placeholder/" : {
+      "get" : {
+        "tags" : [ "media" ],
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "labels",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
```

Comment:
for the parameters, we would want to be passing an image in the body of the request - [this is the relevant bit of the docs](https://swagger.io/docs/specification/describing-request-body/file-upload/)

Reply:
image_classification field type changed